src/app/common/utils-view.ts

```diff
@@ -71,6 +72,12 @@ export class SectionHeader implements VirtualDOM<'header'> {
      * @group Immutable Constants
      */
     public readonly icon: string
+    /**
```

Comment:
Here we are in `common`, it should not include something specific to `Environment` or `Projects` or `CDN`, or `System` . The onclick seems also wrong, a better way to inject what we want is needed. 
What you can do is to add in the constructor a parameters   `withChildren: AnyVirtualDOM[]`  that is expanded at the end of the children list.